docs/index.md

```diff
@@ -15,6 +15,7 @@ Example inference speed (RTX 4090, Ryzen 9 7950X, 64 tokens):
 - Install: `pip install autoawq`.
 - Your torch version must match the build version, i.e. you cannot use torch 2.0.1 with a wheel that was built with 2.2.0.
 - For AMD GPUs, inference will run through ExLlamaV2 kernels without fused layers. You need to pass the following arguments to run with AMD GPUs:
+- For CPU device, you should install intel-extension-for-transformers with `pip install intel-extension-for-transformers`. And torch 2.2 version is required since "intel-extension-for-transformers" was built with torch 2.2. And you should use "use_qbits=True" for CPU device. Up to now, the feature of fuse_layers hasn't been ready for CPU device.
```

Comment:
we don't explicitly indicate should use torch2.2+cpu in the requirements.txt of ITREX, we just add the `--extra-index-url https://download.pytorch.org/whl/cpu` to get torch CPU version.
looks like current ITREX always try to get the latest torch version when build from source.
should we indicate the torch version which we link against in our release log?

Reply:
or let user build itrex from source when their torch not match with the torch version which itrex link against?  :thinking: